evals/benchmark/benchmark.yaml

```diff
@@ -67,8 +68,15 @@ test_cases:
       service_list:  # Replace with your k8s service names if deploy with k8s
                      # or container names if deploy with Docker for metrics collection,
                      # activate if collect_service_metric is true
-        - "chatqna-tei"
-        - "chatqna-teirerank"
+        - "chatqna-backend-server-svc"
+        - "chatqna-nginx-svc"
+        - "dataprep-svc"
+        - "embedding-dependency-svc"
+        - "llm-dependency-svc"
+        - "reranking-dependency-svc"
+        - "retriever-svc"
+        - "vector-db"
```

Comment:
The suffix '-svc' does not represent the k8s service that manages the network. What manifest parameters are they related to?